src/components/ApplicationToolbar/ApplicationToolbar.tsx

```diff
@@ -14,6 +19,19 @@ type ApplicationToolbarProps = {
 export const ApplicationToolbar = ({ toggle }: ApplicationToolbarProps) => {
   const classes = useStyles();
 
+  const dispatch = useDispatch();
+
+  const [value, setValue] = React.useState<number>(1);
+
+  const handleChange = (event: any, newValue: any) => {
```

Comment:
use `onChange` and type both `event` and `newValue`. You should also use a more descriptive name than `newValue`.